Correction of Various Documentation Issues

RKStrand opened the pull request:
Various commits to .tex and .rst files to correct issues with the documentation noted by a variety of defect issues.  The one code change is the correction of a comment line only.  This addresses the following defects:

- #10605 
- #10606 
- #10833 
- #10835 
- #10674 
- #10364 

Pull request overview
---------------------

- Fixes #10605 plus others (see list above)

### Description of the purpose of this PR

The purpose of this PR is to correct various documentation errors: typos, improper placement of text, missing sections, etc.  As this is a documentation only PR, there are no unit tests included with it.

### Pull Request Author

<!-- Add to this list or remove from it as applicable.  This is a simple templated set of guidelines. -->

 - [X] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [X] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer

<!-- This will not be exhaustively relevant to every PR. -->

 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

RKStrand committed:
Commits for #10605

Various commits to .tex and .rst files to correct issues with the documentation noted by a variety of defect issues.  The one code change is the correction of a comment line only.

RKStrand commented:
This is ready for some review.  I have built the readthedocs html files for the docs that changed locally and the corrections I made seem to be good.  Of course, all of the IDFs that changed were also built locally and seem ready also.  If there are any issues, please let me know and I'll be happy to make corrections.  Thanks!

Myoldmopar commented on doc/readthedocs/sphinx/auxiliary-programs/auxiliary-programs.rst:
```diff
@@ -2292,17 +2292,17 @@ These are followed by groupings of Monthly temperature data.
-     7:01- 8:00     59    49    46    50    44    31    50    46    52    40    49    38 
-     8:01- 9:00     62    51    50    46    40    29    43    36    47    43    48    43 
-     9:01-10:00     61    53    43    41    37    26    37    32    34    38    43    44 
+                  Jan   Feb   Mar   Apr   May   Jun   Jul   Aug   Sep   Oct   Nov   Dec   
```
Is the data actually coming out like this?  I don't really have a strong opinion about adjusting this text, but I wouldn't want to mislead someone who might be parsing the output.

RKStrand replied:
I'll double check this.  I did look at an existing .stat file as a guide, but I'll take another look and fix what is still off.

RKStrand replied:
It took a couple of attempts to figure a few things out, but I think this is now cleaned up and looking much better now.

Myoldmopar commented on doc/readthedocs/sphinx/auxiliary-programs/auxiliary-programs.rst:
```diff
@@ -938,7 +938,7 @@ following three tables illustrate the names for data elements.
   +----------------+----------------+----------------+----------------+
   | horirsky       | horizon        | W              | Y              |
-  |                | tal_infrared\_ | h m\ :sup:`−2` |                |
+  |                | tal_infrared_  | h m\ :sup:`−2` |                |
```
I think this table still needs love.  Would you want to give this another pass?  https://energyplus.readthedocs.io/en/10605documentationissues2024b/auxiliary-programs/auxiliary-programs.html#id68

If not, I can.  The table is really just too skinny.  I would try to use a tool like https://www.tablesgenerator.com/text_tables to paste in the data and have it generate a nice fully formed version.

RKStrand replied:
I can look at this further and fully clean this table up.

RKStrand replied:
I've cleaned this and I think all of the other tables in this area up.  Shouldn't be any stray spaces or other "glitches" in these tables now.

Myoldmopar reviewed (commented):
I think I'd like to see at least the one table I commented on cleaned up just a little more while we're already in there.  The full set of markdown docs from this branch are shown here: The docs from this branch are built here https://energyplus.readthedocs.io/en/10605documentationissues2024b/, and that should be updated after any commits to this branch.  So if you want to test out more changes, just push your commit, give it 5 minutes to build, and refresh that page.

RKStrand committed:
Additional corrections in response to review

Additional changes were made to the Auxiliary Programs document, particularly in the Weather section.

RKStrand committed:
More touch-up fixes

Still getting the hang of working with this...obviously!

RKStrand committed:
A few more tweaks

Getting closer...this might be it?

Myoldmopar commented on doc/readthedocs/sphinx/auxiliary-programs/auxiliary-programs.rst:
```diff
@@ -901,109 +901,80 @@ following three tables illustrate the names for data elements.
+  | liq_precip_depth  | liquid_precip_depth                              | mm               | Y          |
+  +-------------------+--------------------------------------------------+------------------+------------+
+  | liq_precip_rate   | liquid_precip_rate                               | h                | N          |
+  +-------------------+--------------------------------------------------+------------------+------------+
```
Much better, thanks!
